version.txt

```diff
@@ -1 +1 @@
-4.1.0
+4.2.0
```

Comment:
I think we should bump to `4.1.1` as a minor upgrade would imply there's some sort of new feature.

Reply:
Works for me